lips/lip-6.mdx

```diff
@@ -109,14 +108,11 @@ More on these transaction scripts (their specifications, information about error
 
 This section details the workflows or common scenarios related to Validator Set management.
 ## Validator Owner and Validator Operator accounts creation
-LibraRoot (through the Association Operation Services, "AOS") creates Validator Owner and Validator Operator accounts.
-To get an account created, each Validator Owner or Operator needs to provide an authentication key and a name of their organization to LibraRoot. The authentication key is a 32 elements byte array, derived from the public verification key of Ed25519 or MultiEd25519 signature schemes.
+LibraRoot (through the Association Operation Services, "AOS") creates Validator Owner and Validator Operator accounts. To get an account created, each Validator Owner or Operator needs to provide an authentication key and a name of their organization to LibraRoot. The authentication key is a 32 byte array, derived from the public verification key of Ed25519 or MultiEd25519 signature schemes.
```

Comment:
above, "create_validator_account" change ':' to dash to match style of other scripts